src/components/Header/Header.scss

```diff
@@ -158,33 +164,265 @@
+            overflow: hidden;
+
+            .menu-overlay {
+            overflow: hidden;
+            position: fixed;
+            top: 0;
+            right: -1360px;
+            width: 100%;
+            height: 100%;
+            background-color: rgba(0, 0, 0, 0.1);
+            transition: right 0.5s ease;
             }
-            .responsive-header{
+
+            .menu-overlay.open {
+                right: 0; 
+            }
+                  
+            @keyframes slideIn {
+                0% {
+                  right: -1360px; 
+                }
+                100% {
+                  right: 0; 
+                }
+            }
+                  
+            .menu-overlay.slide-in {
+                animation: slideIn 0.5s ease; 
+            }
+                
+            .dropdown-menu-header{
+                display: flex;
+                justify-content: space-between;
+                width: 100%;
+                align-items: center;
+
+                >:nth-child(1){
+                    width: 175px;
+                }
+                >:nth-child(2){
+                    border: 2px solid #FFC600;
+                    color: #FFC600;
+                    height: 45px;
+                    width: 45px;
+                    font-size: 35px;
+                    font-weight: 600px;
+                    border-radius: 10px;
+                    background: transparent;
+                    z-index: 3;
+                }
+                    
+            }
+            .mobile-menu-text{
+                display: flex;
+                justify-content: center;
+                width: 100%;
+                height: auto;
+                border-bottom: 2px solid #FFC600;
+                padding-bottom: 40px;
+
+                ul {
+                    gap: 30px;
+                    display: flex;
+                    flex-direction: column;
+                    list-style-type: none;
+                    margin: 0;
+                    padding: 0;
+                    text-align: center;
+
+                    li {
+                        font-size: 24px;
+                        font-weight: 600;
+                    }
+
+                    li:hover {
+                        cursor: pointer;
+                    }
+                }
+            }
+            .social-icons {
                 display: flex;
-                gap: 18px;
                 justify-content: center;
                 align-items: center;
+                gap: 20px;
+                margin-bottom: 15px;
+            }
+        }
+
+        .dropdown-menu.open {
+            right: 0; 
+        }
+              
+        @keyframes slideIn {
+            0% {
+              right: -1360px; 
+            }
+            100% {
+              right: 0; 
+            }
+        }
+              
+        .dropdown-menu.slide-in {
+            animation: slideIn 0.5s ease; 
+        }
+    }
+}
+@media screen and (max-width: 460px){
```

Comment:
Sve isto vazi i za ovu velicinu kao i gore samo sto ne znam zasto si ponavljao isti styling kad vec imas sve to za `.dropdown-menu`?

Reply:
Ovde samo dodas style za tu velicinu ako si promenio recimo neki fontSize nema sta da dupliras ceo css

Reply:
Obrisan sav visak, ostavljen samo deo gde smanjujem logo, velicinu dumgmenceta, font i gap na ikonice.